apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -358,7 +358,7 @@ export const BaseTicketForm: React.FC<ITicketFormProps> = (props) => {
                                 {PromptHelpMessage}
                             </Typography.Paragraph>
                         </Prompt>
-                        <Col lg={13} md={24}>
+                        <Col span={15}>
```

Comment:
reason of this bug is incorrect usage of focus container in this solution u still can reproduce this bug